pyblazing/pyblazing/apiv2/context.py

```diff
@@ -3090,6 +3134,27 @@ def sql(
                         query_table.input = [query_table.input]
                     currentTableNodes.append(query_table)
 
+            elif (
+                query_table.fileType == DataType.MYSQL
+                or query_table.fileType == DataType.SQLITE
+                # or query_table.fileType == DataType.
+            ):
+                if query_table.has_metadata():
```

Comment:
This block of code was just copied form elsewhere. It does work, but its misleading and confusing. I would recommend replacing it with the following, which would be more explicit about what its doing:

```
currentTableNodes = []
for i in range(0, numSlices):
                bt = BlazingTable(self.name, self.input, self.fileType, args=self.args)
                bt.column_names = self.column_names
                bt.column_types = self.column_types
                currentTableNodes.append(bt)
```

Reply:
I would apply this changes in the up coming refactor project for pyblazing, as you said the code works but the style can improve, how about we push this suggestion for the next PRs cc @drabastomek